functions/filter/inlet_title_generator.py

```
@@ -0,0 +1,453 @@
+            response_message.get("content")
+            or response_message.get("reasoning_content")
+            or message.get("content", user_message)
+        )
+    else:
+        title_string = ""
+
+    title_string = title_string[title_string.find("{") : title_string.rfind("}") + 1]
+
+    try:
+        title = json.loads(title_string).get("title", user_message)
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)